web/server/codechecker_server/migrations/report/versions/c3dad71f8e6b_store_information_about_enabled_and_disabled_checkers_for_a_run.py

```diff
@@ -214,6 +214,10 @@ def _print_progress(batch: int):
                      if batch < num_batches else report_count,
                      float(batch) / num_batches * 100)
 
+        fake_chk_id = db.execute(
+            "SELECT id FROM checkers WHERE "
+            f"analyzer_name = '{FakeChecker[0]}' AND "
+            f"checker_name = '{FakeChecker[1]}';").scalar()
```

Comment:
```suggestion
        fake_chk_id = db.query(Checker.id).filter(
            Checkers.analyzer_name == FakeChecker[0],
            Checkers.checker_name == FakeChecker[1]
        ).scalar()
```

Reply:
@cservakt You can not use this construct here, because we are in a migration context.

Reply:
Nevermind, this one query (and its corresponding counterpart in `downgrade()`) I was able to refactor.

The bigger queries that deal with inner queries, pagination, UPDATE...SET...FROM are unlikely to be refactorable to something SQLAlchemy can stomach.